UnityProject/Assets/Scripts/HealthV2/Living/PlayerHealthV2.cs

```diff
@@ -65,7 +65,45 @@ private void OnPlayerConsciousStateChangeServer(ConsciousState oldState, Conscio
 		{
 			if (isServer)
 			{
-				playerNetworkActions.OnConsciousStateChanged(oldState, newState);
+				switch (newState)
+				{
+					case ConsciousState.CONSCIOUS:
+						playerMove.ServerAllowInput.RemovePosition(this);
+						playerMove.CurrentMovementType = MovementType.Running;
+						break;
+					case ConsciousState.BARELY_CONSCIOUS:
+						//Drop hand items when unconscious
+						foreach (var itemSlot in playerScript.DynamicItemStorage.GetHandSlots())
+						{
+							Inventory.ServerDrop(itemSlot);
+						}
+						playerMove.ServerAllowInput.RemovePosition(this);
+						playerMove.CurrentMovementType = MovementType.Running;
+						if (oldState == ConsciousState.CONSCIOUS)
+						{
+							//only play the sound if we are falling
+							SoundManager.PlayNetworkedAtPos(CommonSounds.Instance.Bodyfall, transform.position, sourceObj: gameObject);
```

Comment:
The sounds of falling are actually a bit different between species, last time I checked on SS13. Can you leave a way to change these or should we leave for later once we have the noise?

Reply:
I think the later, since we want to make it dynamic